CMakeLists.txt

```diff
@@ -89,9 +89,15 @@ else ()
   #   interfaces, etc.
   # disable -Wpointer-arith: this is a GCC extension, and doesn't work in MSVC.
   add_definitions(
-    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g
+    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g -std=c++11
```

Comment:
1. We don't build w/ appveyor on VS2013, so it's probably broken anyway.
3. I don't really want to move to anything later than 4.8.4 since that's what I have on my laptop. :)